packages/x-codemod/src/v6.0.0/data-grid/rename-linkOperators-logicOperators/actual.spec.js

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react'
```

Comment:
Could there be other ways (apart from those mentioned in this file) in which breaking changes from [this PR](https://github.com/mui/mui-x/pull/7310) could be used 🤔

Reply:
Unlikely but feasible: If they have a custom filter panel, the link operator will be in the props they reuse

Reply:
Ah ok, that could be one of the use cases, thanks for the input.

I think as long as the other component is defined in the same file it should work with props even, [added](https://github.com/mui/mui-x/pull/7707/commits/d3617d9c80c4e1f8793f401bde088c6d92709bd3) another snippet to validate that.

But the one defined in another file is a limitations of the codemods and it must be handled manually by users.